packages/components/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### New Feature
 
 -   `Flex` and its companion components have been re-written from the ground up and the `isReversed` prop has been deprecated.
+-   Two new layout components, `HStack` and `VStack`, have been added under the experimental namespace.
```

Comment:
Experimental features by design should not be documented. We should either ignore them here or expose as official APIs.